### PR TITLE
Status coverage bug

### PR DESCRIPTION
**Pull request by DanielElisenberg**
# STATUS COVERAGE BUG

The validator only inserts the temporalStatusDates but not the temporal coverage for STATUS type datasets. Looking at the existing data in 1.5 status datasets have both the temporalStatusDates AND temporalCoverage:
```json
{
    "name": "BEFOLKNING_BYDEL",
    "populationDescription": "Variabelen omfatter personer bosatt i kommunene som har bydelsinndeling i offisiell statistikk (Oslo, Bergen, Stavanger, Trondheim).",
    "temporality": "STATUS",
    "temporalCoverage": {
        "start": 7305,
        "stop": 18993
    },
    "temporalStatusDates": [
       14245,
       10227,
       15706,
       13514,
       9131,
```

The validator must supply both to the rest of the pipeline.

~~Bumped version from 6.0.0 => 6.1.0 as this does not affect the API or the user, but is an important bugfix for the usage in our backend.~~

### CODE REVIEW

**Comment by DanielElisenberg** [comment]
Added extra tests to make sure the inserted unit_type_variables are as thoroughly tested as the users input data

File: tests/resources/components/unit_type_variable_schema.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+            "type": "string",
+            "enum": ["STRING", "LONG", "DATE", "DOUBLE"]
+        },
+        "format": {"type": "string"},
+        "unitType": {
+            "type": "object",
+            "properties": {
+                "shortName": {"type": "string"},
+                "name": {"$ref": "#/definitions/multilingualStringType"},
+                "pseudonymized": {"type": "boolean"},
```
Comment (pawbu) [inline tests/resources/components/unit_type_variable_schema.json:155]: With this name, the code in the pipeline would read `if (pseudonymized) then pseudonymize().` I think this name could indicate that pseudonymization is not yet done, e.g. "requiresPseudonymization".

**Review by pawbu — APPROVED** [review]
Looks good, one important question

File: tests/resources/components/unit_type_variable_schema.json (hunk 1 above)
Comment (DanielElisenberg) [reply tests/resources/components/unit_type_variable_schema.json:155]: This is sensible 👍🏻 Will change!

**Pull request merged**